spec/features/mentors/applications/index_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+
+RSpec.describe 'Application index page', type: :feature do
+  let(:user)         { create(:user) }
+  let(:project)      { create(:project, :in_current_season, :accepted, submitter: user) }
+  let(:team1)        { create(:team, name: 'We could be Heroines') }
+  let(:team2)        { create(:team, name: 'Electric Queen') }
+  let(:application1) { create(:application, :in_current_season, :for_project, project1: project, team: team1) }
+  let(:application2) { create(:application, :in_current_season, :for_project, project2: project, team: team2) }
+
+  # TODO: use timecop to make sure the site is available during season
+
+  context 'when mentoring a single projcet' do
+    before do
+      application1
```

Comment:
Turn these into `let!`

Reply:
💡 having solely worked on projects where Rubocop discourages the use of `let!` I actually forgot about it 🙈